home/setup-client-windows.html

```diff
@@ -59,7 +59,7 @@ <h6>Game - Update Steps (manual)</h6>
 <ul>
   <li>Browse using “File Explorer” and go to the folder that you installed FFXI into (as mentioned above, we'll use <strong>C:\PlayOnline\</strong>).  Go into the SquareEnix sub-folder followed by the FINAL FANTASY XI sub-folder (for full clarity, again using the above folder as our example, the path to the folder you should be at would be <strong>C:\PlayOnline\SquareEnix\FINAL FANTASY XI</strong></li>
   <li>Create a new file called file.txt</li>
-  <li>Open file.txt with some type of text editor, such as Windows Notepad or Notepad++ and populate it with the info below. <mark class="pen-red">Make sure to also add a 3rd empty line.</mark>.</li>
+  <li>Open file.txt with some type of text editor, such as Windows Notepad or Notepad++ and populate it with the info below. <strong></strong><mark class="pen-red">Make sure to also add a 3rd empty line!</mark></strong></li>
```

Comment:
why is here an opening and closing of strong without any content?

Reply:
Feexed! Thanks for catching that!